src/Sarif.UnitTests/Converters/FxCopConverterTests.cs

```diff
@@ -188,12 +188,13 @@ public void FxCopLogReader_Context_RefineMessage()
         {
             var context = TestHelper.CreateProjectContext();
 
-            context.RefineMessage("CA0000", "VeryUsefulCheck", "1", "MyCategory", "Breaking");
+            context.RefineMessage("CA0000", "VeryUsefulCheck", "1", "MyCategory", "Breaking", "ExcludedInSource");
```

Comment:
I thought we had names `"Unknown"`, `"SuppressedInSource"`, and `"SuppressedInBaseLine"`.